### PR TITLE
Bundle on build

### PR DESCRIPTION
**Pull request by jskeet**
Again, probably best checked one commit at a time. Oh, and it's already live, so I hope it's okay :)

### CODE REVIEW

File: src/NodaTime.Web/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+wwwroot/js/*.min.js
+wwwroot/css/*.min.css
```
Comment (malcolmr) [inline src/NodaTime.Web/.gitignore]: Let's list `site.min.js` and `site.min.css` explicitly; these two currently also match the foundation minified JS/CSS, which we don't want to ignore.

File: src/NodaTime.Web/wwwroot/lib/bootstrap/.bower.json [hunk 2]
```diff
@@ -1,44 +0,0 @@
-{
```
Comment (malcolmr) [inline src/NodaTime.Web/wwwroot/lib/bootstrap/.bower.json]: We aren't actually using any of the Bower-sourced files at all, so we can actually remove all of `wwwroot/lib`, `.bowerrc`, and `bower.json` (the latter still refers to bootstrap).

File: src/NodaTime.Web/.gitignore (hunk 1 above)
Comment (jskeet) [inline src/NodaTime.Web/.gitignore]: Oops, good plan.

File: src/NodaTime.Web/wwwroot/lib/bootstrap/.bower.json (hunk 2 above)
Comment (jskeet) [inline src/NodaTime.Web/wwwroot/lib/bootstrap/.bower.json]: Will do - as I've just noticed, it's restored bootstrap....

**Comment by malcolmr** [comment]
Hm. Does this actually work for you? I'm not actually seeing these files built when I run `dotnet build` (or run) now.

**Comment by jskeet** [comment]
Yes, it definitely works for me. Build output of `dotnet build`:

```text
Microsoft (R) Build Engine version 15.1.548.43366
Copyright (C) Microsoft Corporation. All rights reserved.


  Bundler: Begin processing bundleconfig.json
        Minified wwwroot/css/site.min.css
        Minified wwwroot/js/site.min.js
  Bundler: Done processing bundleconfig.json
  NodaTime -> c:\Users\skeet\Test\Projects\nodatime\src\NodaTime\bin\Debug\netstandard1.3\NodaTime.dll
  NodaTime.TzdbCompiler -> c:\Users\skeet\Test\Projects\nodatime\src\NodaTime.TzdbCompiler\bin\Debug\netcoreapp1.0\NodaTime.TzdbCompiler.dll
  NodaTime.TzValidate.NodaDump -> c:\Users\skeet\Test\Projects\nodatime\src\NodaTime.TzValidate.NodaDump\bin\Debug\netcoreapp1.0\NodaTime.TzValidate.NodaDump.dll
  NodaTime.Web -> c:\Users\skeet\Test\Projects\nodatime\src\NodaTime.Web\bin\Debug\netcoreapp1.0\NodaTime.Web.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:19.80
```

... and the files are definitely regenerated

**Comment by malcolmr** [comment]
Oh, thanks. `dotnet restore` fixed it.

Sorry, I would have expected that to error out if I needed to run that, not just silently ignore me :(

**Pull request merged**